src/components/ArticleById.jsx

```diff
@@ -113,11 +113,22 @@ function ArticleById() {
           </button>
           {isError ? <h4>{isError}</h4> : null}
         </div>
-        <h4 className="article-comment-count">
-          Comments: {article.comment_count}
-        </h4>
       </div>
 
+      <section>
+        <p className="article-comment-and-vote-count">
+          Votes: {votes} Comments: {article.comment_count}
+        </p>
+      </section>
+
+      <section className="comment-box">
+        <AddComment
+          key={99999}
```

Comment:
Id like to see a more appropriate key here, theres technically nothing wrong with it but something like a number isn't always unique in a long term app, i would suggest something like 'add-comment-component.